tools/nightly-builder/build-required.js

```diff
@@ -0,0 +1,49 @@
+"use strict"
+
+const strftime     = require('strftime').timezone(0)
+    , after        = require('after')
+    , latestBuild  = require('./latest-build')
+    , latestCommit = require('./latest-commit')
+
+const dateFormat = '%Y%m%d'
+
+
+function timeString () {
+  return strftime(dateFormat)
+}
+
+
+function buildRequired (type, callback) {
+  let done = after(2, onData)
+    , build
+    , commit
+
+  function onData (err) {
+    if (err)
+      return callback(err)
+
+    let buildCommit = build.commit.substr(0, 10)
+      , buildDate   = strftime(dateFormat, build.date)
```

Comment:
> We could enforce one-nightly-per-day with this but at the moment it's not.

Hard to bother at this point. Anyone that works with the nightlies will already have to deal with more than one a day.